lib/bolt/cli/src/commands/admin/cluster/datacenter.rs

```diff
@@ -71,6 +88,29 @@ pub enum SubCommand {
 		#[clap(long, short = 'd')]
 		datacenter_name_id: String,
 	},
+	Update {
+		/// The name id of the cluster
+		#[clap(long, short = 'c')]
+		cluster_name_id: String,
+		/// The name id of the datacenter
+		#[clap(long, short = 'd')]
+		datacenter_name_id: String,
+		/// The pool type
+		#[clap(long)]
+		pool_type: DatacenterPoolType,
```

Comment:
`pool`

this is a little confusing initially, but whatever let's worry about it later